packages/mobile/src/web3/saga.ts

```diff
@@ -267,6 +224,9 @@ export function* watchRefreshGasPrice() {
 }
 
 export function* web3Saga() {
-  yield spawn(checkSyncProgressWorker)
   yield spawn(watchRefreshGasPrice)
 }
+
+// exported for testing
+export const _checkWeb3SyncProgressClaim = checkWeb3SyncProgressClaim
```

Comment:
what do you think of just renaming `checkWeb3SyncProgressClaim` to `_checkWeb3SyncProgressClaim` and exporting it?